x/derivatives/types/positions.go

```diff
@@ -234,11 +234,14 @@ func (m PerpetualFuturesPosition) RequiredMarginInBase() sdk.Int {
 func (m PerpetualFuturesPosition) RequiredMarginInMetrics(baseMetricsRate, quoteMetricsRate MetricsRateType) sdk.Int {
```

Comment:
I think InMetrics means in "usd" as the comment says.

If you modify it in "uusd" (micro metrics), change the function name.